Header header
geometry_msgs/Pose pose
string label # Represents the incoming labels